gzip.go

```diff
@@ -33,13 +33,28 @@ type NotWorthGzipCompressing interface {
 // It's aware of GzipByter and NotWorthGzipCompressing interfaces, and uses them
 // to improve performance when the provided content implements them. Otherwise,
 // it applies gzip compression on the fly, if it's found to be beneficial.
-func ServeContent(w http.ResponseWriter, req *http.Request, name string, modTime time.Time, content io.ReadSeeker) {
+func ServeContent(fs *fileServer, w http.ResponseWriter, req *http.Request, name string, modTime time.Time, fpath string, content io.ReadSeeker) {
 	// If compression has already been dealt with, serve as is.
 	if _, ok := w.Header()["Content-Encoding"]; ok {
 		http.ServeContent(w, req, name, modTime, content)
 		return
 	}
 
+	// If request accepts Brotli, we look for a precompressed variant of this file.
+	// We do not attempt to dynamically compress Brotli as it is not performant.
+	// Example: If `index.js` is requested and `index.js.br` is found, we serve the compressed version.
```

Comment:
Nit: this is an implementation detail of `findBrotliFile`. I'd either move that comment there, or omit it entirely, because there is a risk that the implementation changes and this comment becomes deceiving. (Admittedly unlikely in this case, but in the general case it's a risk.)